ci/test.sh

```diff
@@ -30,6 +30,10 @@ markEndOfBlock "Code Generator Testing"
 
 markStartOfBlock "Editmode Testing"
 
+if [ -d "${PROJECT_DIR}/workers/unity/Library" ]; then
```

Comment:
What's this for?

Reply:
Making a backup of the `Library` directory when you run tests so it can be restored afterwards, otherwise it resets the window layout every time you run `test.sh` and it was annoying me quite a bit, can be moved into its own case